setools/infoflow.py

```diff
@@ -33,7 +35,7 @@ class InfoFlowAnalysis:
 
     """Information flow analysis."""
 
-    def __init__(self, policy, perm_map, min_weight=1, exclude=None):
+    def __init__(self, policy, perm_map, min_weight=1, exclude=None, default_tunable=False):
```

Comment:
I don't like the default_tunable kwarg, though I don't immediately have a better suggestion.

This should probably have a more complex interface, so that it can handle the following scenarios:
1. no filtering of conditionals
1. use default policy settings and filter disabled rules
1. use provided Boolean values and filter disabled rules

It shouldn't be too complicated to implement, since the `evaluate()` method does the heavy lifting already, you just have to give it the information.

References to "tunable" should be replaced with "conditional".